packages/react/src/components/Tooltip/TooltipHost.base.tsx

```diff
@@ -114,15 +112,13 @@ export class TooltipHostBase extends React.Component<ITooltipHostProps, ITooltip
             })}
             onMouseEnter={this._onTooltipMouseEnter}
             onMouseLeave={this._onTooltipMouseLeave}
-            {...getNativeProps(this.props, divProperties)}
+            {...getNativeProps(this.props, divProperties, ['id'])} // Make sure we use the id above
             {...tooltipProps}
           />
         )}
-        {isAriaPlaceholderRendered && (
-          <div id={tooltipId} role="none" style={hiddenContentStyle as React.CSSProperties}>
-            {content}
-          </div>
-        )}
+        <div id={tooltipId} role="none" style={hiddenContentStyle as React.CSSProperties}>
```

Comment:
I tried marking this element `aria-hidden="true"` to avoid the issue where this can be focused by a virtual cursor and read twice. This worked for me in VO and Narrator but NVDA would not read the tooltips in this case.